source/_docs/configuration.markdown

```diff
@@ -40,6 +40,7 @@ If you can't see your integration listed there, you will need to restart Home As
 
 ## Migrating to a new system
 
-The preferred way of migrating to a new system is by {% my supervisor_backups title="making a backup" %}.
+The preferred way of migrating to a new system is by backup using {% my supervisor_backups title="supervisor backup" %}. Then install new system (HAOS or Supervisor), and restore backup at first logon.
 
-If you run the container or core installation methods, you will need to manually make a backup of your configuration folder. Be aware that some of the files you need start with `.`, which is hidden by default from both `ls` (in SSH), in Windows Explorer, and macOS Finder. You'll need to ensure that you're viewing all files before you copy them.
+If you run the container or core installation method you don't have supervisor-backup and will need to use the {% my backup title="backup-integration" %} to make a backup.
```

Comment:
I now see the problem you are trying to solve with the "supervisor backup". Maybe instead, we should write something like:

> The preferred way of migrating to a new system is by making a backup. Backups can be found and created in the Settings menu, next go to **System** -> **Backup**.

The my links might be more confusing, so maybe just leaving them out, in this case, is an option?